templates/page/page.tpl.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * @file
+ * Returns the HTML for a single Drupal page.
+ *
+ * Complete documentation for this file is available online.
+ * @see https://drupal.org/node/1728148
+ */
```

Comment:
It'd be great to add documentation for the various variables and regions that are used in this file. See [Bartik](https://www.drupal.org/project/bartik)'s `page.tpl.php` file for a good base example.